### PR TITLE
Cell

### PR DESCRIPTION
**Pull request by jimmacur**
This completes iteration 1. Render method now clean and ordered correctly.

### CODE REVIEW

File: lib/cell.rb [hunk 1]
```diff
@@ -28,14 +28,14 @@ def fire_upon
...
+        elsif @fired_upon && @ship
             "H"
         elsif @fired_upon
             "M"
         elsif player_board && @ship
             "S"
-        elsif @ship.sunk?
-            "X"
         else
             "."
```
Comment (jchirch) [inline lib/cell.rb:40]: Woah nice job with the render method! The logic looks solid and tracks. I like the changed parameter to player_board.

File: lib/cell.rb [hunk 2]
```diff
@@ -22,20 +22,20 @@ def fired_upon?
 
     def fire_upon
         @fired_upon = true
-        if @ship != nil
+        if @ship
```
Comment (jchirch) [inline lib/cell.rb:25]: As much as I liked the `!= nil` logic, this makes more sense. RIP `!= nil`

**Comment by jchirch** [comment]
Great work Jim! This problem definitely spoke to you and I'm glad you were able to crack it.

File: lib/cell.rb (hunk 1 above)
Comment (jimmacur) [inline lib/cell.rb:40]: Can you see this comment?

Comment (jchirch) [reply lib/cell.rb:40]: 10 4 good buddy

**Review by jchirch — COMMENTED** [review]
test

**Pull request merged**